modules/pipeline/aop/plugins/pipeline/testplan_after/plugin.go

```diff
@@ -188,3 +172,38 @@ func convertReport(pipelineID uint64, report spec.PipelineReport) (ApiReportMeta
 	}
 	return meta, nil
 }
+
+func (p *provider) sendMessage(req testplanpb.Content, configString string) error {
+	// get projectID from GlobalConfig
+	config := apistructs.AutoTestAPIConfig{}
+	err := json.Unmarshal([]byte(configString), &config)
+	if err != nil {
+		return err
+	}
+
+	projectID, err := strconv.ParseUint(config.Header["project-id"], 10, 64)
+	project, err := p.Bundle.GetProject(projectID)
+	if err != nil {
+		return err
+	}
+
+	ev2 := &apistructs.EventCreateRequest{
+		EventHeader: apistructs.EventHeader{
+			Event:         bundle.AutoTestPlanExecuteEvent,
+			Action:        bundle.UpdateAction,
+			OrgID:         fmt.Sprintf("%d", project.OrgID),
```

Comment:
why not get org-id from header, why we need query project entity by bundle?

Reply:
![image](https://user-images.githubusercontent.com/74653472/133053851-da099194-be1a-4d6f-be1f-e04107bb2871.png)
Pipeline.Labels only hava key:has-report-auto-test-execute-config value:true.
in execute config, org_id is null, only have projectID

Reply:
I will try to set projectID and orgID when create pipeline

Reply:
done